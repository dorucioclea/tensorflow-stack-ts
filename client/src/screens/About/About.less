.about {
  .about-header {
    .about-logo {
      animation: about-logo-spin 10s linear;
      height: 80px;
    }
  }
}

@keyframes about-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}