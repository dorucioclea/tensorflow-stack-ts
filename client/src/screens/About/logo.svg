<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="308px" height="292px" viewBox="0 0 3080 2920" preserveAspectRatio="xMidYMid meet">
<g id="layer102" fill="#26a41e" stroke="none">
 <path d="M1499 2766 c-2 -2 -44 -7 -94 -11 -108 -8 -125 -10 -146 -19 -9 -4 -20 -6 -25 -5 -5 1 -16 -1 -24 -4 -8 -4 -26 -8 -40 -10 -14 -2 -33 -7 -42 -11 -9 -4 -25 -9 -35 -11 -10 -2 -19 -4 -20 -5 -2 -2 -7 -4 -13 -5 -5 -1 -12 -4 -15 -5 -3 -1 -8 -3 -12 -4 -5 -1 -23 -10 -40 -19 -18 -10 -36 -18 -40 -18 -21 -3 -44 -16 -120 -66 -45 -29 -87 -53 -93 -53 -6 0 -13 -7 -16 -15 -4 -8 -11 -15 -17 -15 -12 0 -10 2 -129 -115 -49 -46 -88 -89 -88 -94 0 -6 -9 -15 -20 -21 -11 -6 -20 -17 -20 -24 0 -8 -7 -16 -15 -20 -8 -3 -15 -10 -15 -16 0 -6 -25 -49 -55 -95 -30 -45 -55 -87 -55 -93 0 -5 -12 -36 -27 -68 -14 -33 -28 -65 -29 -72 -1 -8 -6 -21 -10 -30 -4 -9 -8 -20 -9 -24 -2 -5 -3 -9 -4 -10 -2 -2 -4 -14 -7 -28 -2 -14 -7 -36 -11 -50 -3 -14 -7 -29 -8 -35 -1 -5 -3 -12 -4 -15 -19 -44 -19 -495 0 -495 1 0 3 -9 5 -20 2 -11 7 -37 12 -57 6 -21 8 -38 5 -38 -3 0 0 -7 7 -15 7 -8 9 -15 5 -15 -4 0 -3 -6 3 -13 6 -7 13 -24 16 -37 2 -14 16 -49 30 -77 14 -29 26 -59 26 -67 0 -7 8 -24 18 -37 10 -13 22 -31 25 -39 11 -23 59 -93 124 -179 23 -30 164 -171 194 -194 66 -50 116 -84 150 -103 21 -12 50 -31 65 -43 15 -12 33 -21 40 -21 8 0 38 -12 67 -26 28 -14 63 -28 77 -30 13 -3 30 -10 37 -16 7 -6 13 -7 13 -3 0 4 7 2 15 -5 8 -7 15 -10 15 -7 0 3 17 1 38 -5 20 -5 46 -10 57 -12 11 -2 20 -4 20 -5 0 -19 451 -19 495 0 3 1 10 3 15 4 6 1 21 5 35 8 14 4 36 9 50 11 14 3 26 5 28 7 1 1 5 2 10 4 4 1 15 5 24 9 9 4 22 9 30 10 7 1 39 15 72 29 32 15 63 27 68 27 6 0 49 26 96 57 48 32 91 56 95 55 4 -1 10 5 13 13 4 8 12 15 20 15 7 0 18 9 24 20 6 11 15 20 21 20 5 0 48 39 94 88 117 120 115 117 115 130 0 7 4 12 9 12 4 0 30 37 57 81 71 118 94 159 88 159 -2 0 3 15 13 33 9 17 18 35 19 40 1 4 3 9 4 12 1 3 4 10 5 15 1 6 3 11 5 13 1 1 3 10 5 20 2 10 7 26 11 35 4 9 9 28 11 42 2 14 6 32 10 40 3 8 5 19 4 24 -1 5 1 16 5 25 9 20 11 37 20 146 9 111 9 102 0 210 -9 108 -11 125 -20 146 -4 9 -6 20 -5 25 1 5 -1 16 -4 24 -4 8 -8 26 -10 40 -2 14 -7 33 -11 42 -4 9 -9 25 -11 35 -2 10 -4 19 -5 21 -2 1 -3 5 -4 10 -1 4 -5 13 -9 20 -5 6 -5 12 -2 12 4 0 -2 15 -13 33 -12 17 -21 34 -21 37 0 16 -25 66 -47 96 -15 19 -23 34 -19 34 4 0 -5 14 -19 31 -14 17 -26 37 -26 44 0 8 -9 20 -20 28 -11 8 -18 16 -15 19 7 6 -212 228 -225 228 -5 0 -10 4 -10 9 0 11 -82 72 -92 68 -3 -1 -18 9 -32 22 -14 14 -26 21 -26 17 0 -5 -15 3 -34 18 -32 24 -74 45 -86 42 -3 0 -22 10 -42 22 -21 13 -38 20 -38 17 0 -4 -7 -1 -17 6 -9 8 -18 13 -20 12 -1 -2 -4 -3 -5 -2 -2 0 -11 2 -21 4 -10 2 -26 7 -35 11 -9 4 -28 9 -42 11 -14 2 -32 6 -40 10 -8 3 -19 5 -24 4 -5 -1 -16 1 -25 5 -22 9 -33 11 -131 18 -47 4 -94 9 -106 11 -12 3 -23 3 -25 1z m166 -146 c0 -1 11 -4 25 -5 36 -3 135 -21 135 -25 0 -1 8 -3 18 -6 9 -2 25 -6 35 -8 9 -3 17 -5 17 -6 0 -1 8 -3 17 -5 9 -2 32 -11 50 -20 18 -9 41 -18 52 -20 10 -3 28 -12 38 -22 11 -10 22 -17 24 -15 11 6 25 -1 65 -33 24 -19 49 -35 55 -35 7 0 14 -7 18 -15 3 -8 11 -15 17 -15 6 0 21 -10 33 -23 11 -12 35 -32 51 -44 l30 -22 -835 0 -835 0 35 29 c95 80 150 123 150 115 0 -4 14 5 31 19 28 24 42 30 53 24 2 -2 13 5 24 15 10 10 28 19 38 22 11 2 34 11 51 20 18 9 45 18 59 21 15 3 35 9 45 15 10 5 31 11 46 13 15 3 42 8 58 11 17 4 45 8 63 9 18 2 34 5 37 7 4 4 300 3 300 -1z m-225 -585 l0 -125 -337 -2 -338 -3 0 -460 0 -460 240 -3 c132 -1 246 -4 254 -7 11 -4 11 -9 -3 -24 -10 -11 -18 -26 -20 -34 -1 -8 -8 -22 -15 -30 -7 -9 -14 -22 -15 -29 -1 -7 -8 -24 -16 -38 -8 -14 -41 -79 -73 -144 -67 -136 -67 -151 1 -178 39 -15 42 -15 58 3 9 10 14 19 10 19 -4 0 2 14 14 30 12 17 18 30 14 30 -3 0 -2 6 4 12 18 23 92 166 92 179 0 7 7 20 16 30 8 9 16 23 18 31 1 7 14 34 30 60 15 26 29 56 30 65 2 16 16 18 95 21 87 3 129 -7 116 -28 -2 -3 3 -18 13 -31 9 -13 17 -30 18 -37 2 -7 8 -20 15 -29 7 -8 14 -22 15 -30 1 -7 8 -21 15 -30 7 -8 14 -22 15 -30 1 -7 8 -21 15 -30 7 -8 14 -22 15 -30 1 -7 8 -21 15 -30 7 -8 14 -21 15 -29 1 -7 12 -27 24 -44 12 -16 19 -30 15 -30 -3 0 4 -18 15 -41 19 -37 24 -40 54 -36 50 7 89 36 81 61 -3 12 -7 26 -8 32 -2 7 -9 19 -16 27 -7 9 -14 23 -15 30 -1 8 -8 22 -15 30 -7 9 -14 23 -15 30 -1 8 -7 22 -14 30 -7 8 -14 23 -16 34 -2 10 -27 62 -55 116 l-51 97 230 0 c126 0 231 4 234 9 3 5 6 213 6 462 l1 454 -321 3 -320 2 0 125 0 125 437 0 436 0 30 -32 c16 -18 36 -48 44 -67 8 -19 26 -50 40 -68 14 -18 22 -33 19 -33 -4 0 0 -10 8 -22 8 -13 15 -25 15 -28 0 -3 5 -18 12 -35 7 -16 13 -33 14 -37 2 -5 4 -10 5 -13 1 -3 4 -9 5 -15 1 -5 3 -11 5 -12 1 -2 3 -11 6 -20 2 -10 6 -26 8 -35 3 -10 5 -18 6 -18 7 0 30 -127 25 -135 -3 -5 -2 -11 3 -14 6 -4 9 -291 2 -291 -1 0 -4 -13 -5 -30 -2 -16 -6 -43 -10 -60 -3 -16 -8 -41 -10 -55 -2 -14 -12 -43 -21 -65 -9 -22 -18 -51 -21 -64 -2 -14 -9 -31 -14 -37 -5 -7 -12 -22 -14 -34 -2 -13 -12 -31 -21 -41 -10 -11 -13 -19 -9 -19 4 0 0 -9 -10 -20 -10 -11 -15 -20 -11 -20 9 0 -21 -45 -64 -98 -19 -24 -35 -47 -35 -52 0 -4 -12 -18 -27 -30 -16 -12 -28 -27 -28 -35 0 -13 -74 -85 -88 -85 -4 0 -26 -18 -50 -40 -24 -22 -46 -40 -51 -40 -4 0 -18 -11 -31 -25 -12 -13 -26 -22 -31 -20 -4 3 -10 1 -14 -5 -3 -5 -25 -19 -48 -31 -23 -11 -60 -30 -82 -41 -22 -11 -47 -22 -55 -23 -8 -2 -16 -4 -17 -4 -2 -1 -5 0 -6 2 -2 1 -11 -4 -20 -12 -10 -7 -17 -10 -17 -6 0 4 -6 3 -12 -2 -12 -10 -55 -24 -88 -29 -8 -1 -26 -5 -40 -9 -102 -24 -376 -24 -480 0 -47 11 -55 13 -55 15 0 1 -13 4 -30 6 -16 3 -35 10 -42 16 -7 6 -13 7 -13 3 0 -4 -7 -2 -15 5 -8 7 -15 9 -15 5 0 -4 -7 -2 -15 5 -8 7 -15 9 -15 6 0 -6 -54 19 -100 45 -8 5 -26 15 -40 21 -14 7 -39 23 -56 35 -17 13 -34 23 -39 23 -7 0 -24 13 -111 86 -40 34 -88 81 -119 119 -14 17 -38 46 -55 66 -16 20 -30 40 -30 44 0 5 -11 23 -25 41 -14 18 -25 37 -25 42 0 5 -9 21 -21 36 -11 14 -18 26 -14 26 4 0 -4 17 -17 38 -12 20 -21 37 -20 37 2 0 -2 14 -8 30 -7 17 -13 37 -15 45 -2 8 -4 16 -4 18 -1 1 -1 4 -1 7 -1 2 -8 21 -16 41 -8 20 -13 39 -10 41 3 3 1 13 -3 22 -14 26 -25 136 -25 251 0 104 3 135 28 273 2 12 9 34 15 50 6 15 8 27 5 27 -3 0 -2 6 3 13 4 6 8 18 9 25 1 7 12 31 23 52 11 22 18 40 15 40 -2 0 1 10 8 23 7 12 12 24 13 27 0 3 16 29 35 59 19 29 35 57 35 61 0 5 13 24 30 44 l30 36 437 0 438 0 0 -125z m650 -590 l0 -335 -590 0 -590 0 0 335 0 335 590 0 590 0 0 -335z"/>
 <path d="M1234 1626 c-3 -7 -4 -35 -2 -62 l3 -49 270 0 270 0 0 60 0 60 -268 3 c-221 2 -268 0 -273 -12z"/>
 <path d="M1177 1422 c-25 -3 -57 -43 -57 -70 0 -18 49 -62 70 -62 34 0 62 36 58 77 -3 41 -29 61 -71 55z"/>
 <path d="M1787 1400 c-60 -66 30 -152 91 -87 28 29 28 68 2 92 -30 27 -67 25 -93 -5z"/>
 </g>

</svg>