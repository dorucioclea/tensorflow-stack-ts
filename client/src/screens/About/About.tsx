import * as React from "react";
import { Card, Rate } from "antd";
import logo from "./logo.svg";
import "./About.less";

export const About = () => {
  return (
    <Card bordered title="About TensorFlow Stack TS - Kickstart Your AI Project">
      <div className="about">
        <div className="about-header">
          <img src={logo} className="about-logo" alt="logo" />
        </div>
        <p className="about-intro">
          App version: {process.env.REACT_APP_VERSION}<br />
          <a href={`${process.env.REACT_APP_ENDPOINT}`}>GraphQL endpoint</a><br />
          <a href={`${process.env.REACT_APP_HEALTHCHECK}`}>Service Healthcheck</a><br />
          <a href="https://github.com/eram/tensorflow-stack-ts">Project Github</a><br />
          To get started, edit <code>src/containers/pages/About.tsx</code> and in a few seconds it reloads.
        </p>
        <Rate character="6" />
      </div>
    </Card>
  );
};
