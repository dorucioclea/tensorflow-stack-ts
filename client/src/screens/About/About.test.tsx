// Jest snapshot test
import * as React from "react";
import * as TestRenderer from "react-test-renderer";
import { About } from "./About";

describe("About testing", () => {

    test("page is loading correctly", () => {
        const component = TestRenderer.create(
            <About />,
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
