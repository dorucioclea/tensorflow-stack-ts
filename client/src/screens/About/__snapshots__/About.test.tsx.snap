// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About testing page is loading correctly 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        About TensorFlow Stack TS - Kickstart Your AI Project
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="about"
    >
      <div
        className="about-header"
      >
        <img
          alt="logo"
          className="about-logo"
          src="logo.svg"
        />
      </div>
      <p
        className="about-intro"
      >
        App version: 
        <br />
        <a
          href="undefined"
        >
          GraphQL endpoint
        </a>
        <br />
        <a
          href="undefined"
        >
          Service Healthcheck
        </a>
        <br />
        <a
          href="https://github.com/eram/tensorflow-stack-ts"
        >
          Project Github
        </a>
        <br />
        To get started, edit 
        <code>
          src/containers/pages/About.tsx
        </code>
         and in a few seconds it reloads.
      </p>
      <ul
        className="ant-rate"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onMouseLeave={[Function]}
        role="radiogroup"
        style={Object {}}
        tabIndex={0}
      >
        <li
          aria-checked="false"
          aria-posinset={1}
          aria-setsize={5}
          className="ant-rate-star ant-rate-star-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseMove={[Function]}
          role="radio"
          tabIndex={0}
        >
          <div
            className="ant-rate-star-first"
          >
            6
          </div>
          <div
            className="ant-rate-star-second"
          >
            6
          </div>
        </li>
        <li
          aria-checked="false"
          aria-posinset={2}
          aria-setsize={5}
          className="ant-rate-star ant-rate-star-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseMove={[Function]}
          role="radio"
          tabIndex={0}
        >
          <div
            className="ant-rate-star-first"
          >
            6
          </div>
          <div
            className="ant-rate-star-second"
          >
            6
          </div>
        </li>
        <li
          aria-checked="false"
          aria-posinset={3}
          aria-setsize={5}
          className="ant-rate-star ant-rate-star-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseMove={[Function]}
          role="radio"
          tabIndex={0}
        >
          <div
            className="ant-rate-star-first"
          >
            6
          </div>
          <div
            className="ant-rate-star-second"
          >
            6
          </div>
        </li>
        <li
          aria-checked="false"
          aria-posinset={4}
          aria-setsize={5}
          className="ant-rate-star ant-rate-star-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseMove={[Function]}
          role="radio"
          tabIndex={0}
        >
          <div
            className="ant-rate-star-first"
          >
            6
          </div>
          <div
            className="ant-rate-star-second"
          >
            6
          </div>
        </li>
        <li
          aria-checked="false"
          aria-posinset={5}
          aria-setsize={5}
          className="ant-rate-star ant-rate-star-zero"
          onClick={[Function]}
          onKeyDown={[Function]}
          onMouseMove={[Function]}
          role="radio"
          tabIndex={0}
        >
          <div
            className="ant-rate-star-first"
          >
            6
          </div>
          <div
            className="ant-rate-star-second"
          >
            6
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
